GitUI/CommandsDialogs/FormCleanupRepository.cs

```diff
@@ -43,18 +43,37 @@ public void SetPathArgument(string? path)
 
         private void Preview_Click(object sender, EventArgs e)
         {
-            var cleanUpCmd = GitCommandHelpers.CleanCmd(GetCleanMode(), dryRun: true, directories: RemoveDirectories.Checked, paths: GetPathArgumentFromGui());
+            string pathArgument = GetPathArgumentFromGui();
+            CleanMode mode = GetCleanMode();
+            var cleanUpCmd = GitCommandHelpers.CleanCmd(mode, dryRun: true, directories: RemoveDirectories.Checked, paths: pathArgument);
```

Comment:
Please use the explicit type (applies below and to  Cleanup_Click() too).
This PR just add one call in each method, but the type should be clarified.
@mstv and @RussKie should write it down, I often forget myself.

```suggestion
            ArgumentString cleanUpCmd = GitCommandHelpers.CleanCmd(mode, dryRun: true, directories: RemoveDirectories.Checked, paths: pathArgument);
```

Reply:
pushed a change